package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphhopper-js-api-client",
-  "version": "0.9.0-1",
+  "version": "0.9.0-2",
```

Comment:
Unfortunately not. This is not the semantic version standard and is not accepted by npm.